site/docs/guide/introduction.md

```diff
@@ -57,8 +57,6 @@ _________        sends a         ____________                    ____________
         Telegram's responsibility                             your responsibility
 ```
 
-<!-- TODO: create a proper diagram instead of the ASCII art -->
```

Comment:
Is this intended?

Reply:
Yes. At some point I decided that ASCII art is not much worse than images, but it comes with several advantages:
- faster to create
- faster to modify
- needs less tooling
- doesn't require graphics skills from contributors
- works on all screen sizes due to proper scrolling

Reply:
- much easier to translate